src/mail.sh

```diff
@@ -78,17 +78,39 @@ function mail_send()
 {
   local flag="$1"
   local recipients="${options_values['TO']}"
+  local extra_opts="${options_values['PASSTHROUGH']}"
```

Comment:
Maybe we can change `PASSTHROUGH` to something that better portrays that these values will be sent to git-send-email. Maybe `PASS_OPTION_TO_SEND_EMAIL`?